html/status.html

```diff
@@ -489,8 +489,11 @@
 					if (ber > 0)
 						signal += "<div class='level'>BER <progress class='pber progress-1' value='" + ber + "' max='1000' />" + ber + "</div><br />";
 					var ccerr = state['ad_ccerr'][i];
+					var ccerrpre = oldstate['ad_ccerr'][i];
+					ccerrpre = oldstate['ad_ccerr'][i];
```

Comment:
This line is superflous, the variable is already assigned this value on the line above